exporter_test.go

```diff
@@ -238,6 +238,36 @@ func TestTile38(t *testing.T) {
 	}
 }
 
+func TestExportClientList(t *testing.T) {
+	if os.Getenv("TEST_EXPORT_CLIENT_LIST") == "" {
+		t.Skipf("TEST_EXPORT_CLIENT_LIST not set - skipping")
+	}
+
+	for _, isExportClientList := range []bool{true, false} {
+		e, _ := NewRedisExporter(os.Getenv("TEST_EXPORT_CLIENT_LIST"), ExporterOptions{Namespace: "test", ExportClientList: isExportClientList})
```

Comment:
Reuse the exporter that's available via `TEST_REDIS_URI` instead of introducing a new env variable.